source/Ship.cpp

```diff
@@ -2669,6 +2669,19 @@ bool Ship::IsLanding() const
 
 
 
+float Ship::LandingSpeed() const
+{
+	const float mass = Mass();
+	const float landingSpeed = attributes.Get("landing speed");
+	// Ships with a mass under 43 will land at a fixed size,
+	// and those with a mass above will instead progressively land slower.
+	// The landing speed will be applied on top of that, and may be negative
+	// but the ship still should still be able to land at a minimum speed of 0.01f.
```

Comment:
This version seems more understandable, but at least fix the 'size' issue.
```suggestion
	// Ships with mass under 43 will land with fixed speed,
	// and heavier ones will land progressively slower.
	// The landing speed will be applied on top of that, and may be negative,
	// but the ship still should still be able to land at a minimum speed of 0.01f.
```